### PR TITLE
add amp ad intro

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 @jasti PTAL

Preview: https://amp-by-example-staging.appspot.com/amp_ads/introduction/ (demo link doesn't work on staging)

File: src/60_AMP_Ads/Banner_Ad.html [hunk 1]
```diff
@@ -8,43 +8,33 @@
...
+      color: red;
+    }
+  </style>
 </head>
 <body>
-
-  <!-- ## Basic Usage -->
   <!--
-  The anatonomy of banner ad is an image with a hyperlink to the advertiser's page. We display the image for the creative using [amp-img](/components/amp-img/).
+    AMP Ads use the same AMPHTML components as normal AMP pages.
```
Comment (jasti) [inline src/60_AMP_Ads/Banner_Ad.html]: AMP ads are only allowed to use a restricted set of components as described in the spec.

File: src/60_AMP_Ads/Introduction.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!---{
+ "preview": "a4a",
+ "skipValidation": "true"
+}--->
+<!--
+  ## Introduction
+
+  [AMP Ads](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) are based on the AMP runtimne and provide the same benefits as normal AMPHTML files. However, there are a few things that make them different from normal AMPHTML files.
```
Comment (jasti) [inline src/60_AMP_Ads/Introduction.html]: Typo with "AMP Runtime".

Comment (jasti) [inline src/60_AMP_Ads/Introduction.html]: AMP ads actually use a different runtime vs the AMP page. @lannka right?

Comment (lannka) [inline src/60_AMP_Ads/Introduction.html]: Right it's AMP4ADS runtime, a tailored version of AMP runtime.

File: src/60_AMP_Ads/Introduction.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<!doctype html>
+<!-- This tells everyone that this is an AMP for Ads file. -->
+<html ⚡4ads>
+<!-- ### Head -->
+<!-- -->
+<head>
+  <!-- The charset definition must be the first child of the `<head>` tag. -->
+  <meta charset="utf-8">
+  <!-- AMP Ads also require the AMP runtime to be loaded.-->
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
```
Comment (lannka) [inline src/60_AMP_Ads/Introduction.html]: /amp4ads-v0.js

File: src/60_AMP_Ads/Introduction.html (hunk 2 above)
Comment (sebastianbenz) [inline src/60_AMP_Ads/Introduction.html]: ok - then we should change the amp4ad template here: https://validator.ampproject.org/#htmlFormat=AMP4ADS

Also, the validator doesn't complain when using the normal amp runtime. Is this intended?

**Comment by kul3r4** [comment]
I guess we need to disable the validator?

File: src/60_AMP_Ads/Introduction.html (hunk 2 above)
Comment (lannka) [inline src/60_AMP_Ads/Introduction.html]: @Gregable I thought validator is already handling this?

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: src/60_AMP_Ads/_Hello_World.html [hunk 4]
```diff
@@ -28,8 +28,7 @@
   <script async src="https://cdn.ampproject.org/amp4ads-v0.js"></script>
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <!--
-    The AMP Ad boilerplate.
-  -->
+    The AMP Ad boilerplate which is a lot shorter than the regular AMP boilerplate. -->
```
Comment (kul3r4) [inline src/60_AMP_Ads/_Hello_World.html]: which is a lot shorter -> which is shorter
if you say "a lot", not sure how much is "a lot"

File: src/60_AMP_Ads/_Hello_World.html [hunk 5]
```diff
@@ -47,5 +46,7 @@
     Inside the body, AMP ads are only allowed to use a restricted set of HTML and AMPHTML elements as described in the [spec](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md). 
   -->
   <h1>Hello World</h1>
+<!-- Check out the other [AMP Ad samples](/#amp_ads) to learn how to create more sophisticaed AMP Ads.
```
Comment (kul3r4) [inline src/60_AMP_Ads/_Hello_World.html]: typo: sophisticaed

Comment (sebastianbenz) [inline src/60_AMP_Ads/_Hello_World.html]: done

File: src/60_AMP_Ads/_Hello_World.html (hunk 4 above)
Comment (sebastianbenz) [inline src/60_AMP_Ads/_Hello_World.html]: done

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

**Pull request merged**